test/acceptance/features/bindAppToServiceUsingSecret.feature

```diff
@@ -394,3 +394,80 @@ Feature: Bind values from a config map referred in backing service resource
         Then Service Binding "ssd-2" is ready
         And The application env var "BACKEND_USERNAME" has value "AzureDiamond"
         And The application env var "BACKEND_PASSWORD" has value "hunter2"
+
+    Scenario: Verify pulling data from a secret living in another namespace
+        Binding definition is declared on service CRD.
+        Given OLM Operator "backend" is running
```

Comment:
If you use this step, then you should not install CustomResourceDefinition under, or vice versa. I think we need later.

Reply:
Removed OLM step.